packages/logs/README.md

```diff
@@ -117,18 +117,19 @@ window.DD_LOGS.init({
+| Parameter             | Type                                                                      | Required | Default         | Description                                                                                                                                     |
+| --------------------- | ------------------------------------------------------------------------- | -------- | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| `clientToken`         | String                                                                    | Yes      |                 | A [Datadog client token][2].                                                                                                                    |
+| `site`                | String                                                                    | Yes      | `datadoghq.com` | The Datadog site of your organization. US: `datadoghq.com`, EU: `datadoghq.eu`                                                                  |
+| `service`             | String                                                                    | No       |                 | The service name for your application. It should follow the [tag syntax requirements][7].                                                       |
+| `env`                 | String                                                                    | No       |                 | The application’s environment, for example: prod, pre-prod, staging, etc. It should follow the [tag syntax requirements][7].                    |
+| `version`             | String                                                                    | No       |                 | The application’s version, for example: 1.2.3, 6c44da20, 2020.02.13, etc. It should follow the [tag syntax requirements][7].                    |
+| `forwardErrorsToLogs` | Boolean                                                                   | No       | `true`          | Set to `false` to stop forwarding console.error logs, uncaught exceptions and network errors to Datadog.                                        |
+| `forwardConsoleLogs`  | `"all"` or an Array of `"log"` `"debug"` `"info"` `"warn"` `"error"`      | No       | `[]`            | Forward logs from `console.*` to Datadog. Use `"all"` to forward everything or an array of console API names to forward only a subset.          |
+| `forwardReports`      | `"all"` or an Array of `"intervention"` `"deprecation"` `"csp_violation"` | No       | `[]`            | Forward reports from the [Reporting API][8] to Datadog. Use `"all"` to forward everything or an array of report types to forward only a subset. |
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏We should update browser support pages for reporting log/rum events

Reply:
Seems like @amortemousque took care of it already :)
https://github.com/DataDog/browser-sdk/blob/0df00f59308b42ad4273ce9af3c2dba229002ca1/packages/logs/BROWSER_SUPPORT.md#L13-L14
https://github.com/DataDog/browser-sdk/blob/0df00f59308b42ad4273ce9af3c2dba229002ca1/packages/rum/BROWSER_SUPPORT.md#L11-L12

Reply:
Awesome!